ecommerce/mail_api_test.py

```diff
@@ -92,7 +92,7 @@ def test_send_bulk_enroll_emails(mocker, settings):
         assert user_message_props.recipient == assignment.email
         assert user_message_props.context == {
             "enrollable_title": assignment.product_coupon.product.content_object.title,
-            "enrollment_url": "http://test.com/checkout/?product={}&code={}".format(
+            "enrollment_url": "http://test.com/checkout/?is_voucher_applied=False&product={}&code={}".format(
```

Comment:
Can we use f-string here? The linting tools would start complaining about this soon. This can be applied everywhere else too.

Reply:
The existing is better for readability. Ruff hasn't complained about this. If we want to use f-string, we will need to create variables to keep the line length below 88 chars. WDYT?

Reply:
OK. It's fine if ruff isn't complaining about this. But just a thought that we can do a multiline f-string with proper formatting.